gcsweb.k8s.io/ingress.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   annotations:
     kubernetes.io/ingress.global-static-ip-name: gcsweb-k8s-io
 spec:
+  tls:
+  - secretName: gcsweb-k8s-io-tls
```

Comment:
Due to an issue with the way cert-manager obtains certificates from LE (we generate and store a private key *before* the corresponding certificate has been obtained from Let's Encrypt) (more info: https://github.com/jetstack/cert-manager/issues/1343) - if you add this field here when no Secret resource already exists, the ingress-gce ingress controller will fail to actually update the GCP API with the load balancer configuration.

Because of this, you must instead create the Certificate resource and obtain the certificate *before* updating the ingress resource to reference that secret.

We'll be building a workaround into v0.7 which is due in ~1.5wks (most likely, pre-generating a self-signed certificate which can be used in the interim whilst CM is obtaining a publicly trusted certificate).

What this means, is that in order to deploy this *the first time*, you'll need to apply these changes *after* creating all the other resources in this PR 😄

Reply:
I think I discovered this, but I appreciate the clarification. I added a note to our README as well.

Reply:
I've got a fix in progress that will be landing in v0.7 here: https://github.com/jetstack/cert-manager/pull/1392 😄